api/app/src/routes/medical/organization.ts

```diff
@@ -20,22 +25,72 @@ router.post(
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
 
-    // TODO: parse this into model
-    const orgData = req.body;
+    const reqOrgData = organizationSchema.parse(req.body);
+
+    // TODO: I FIGURED THESE 3 WOULD BE GOOD TO STORE IN DB
+    // CAN ADD GATEWAYS OR TECHNICAL CONTACT ETC.
+    const localOrgPayload = {
+      name: reqOrgData.name,
+      type: "Hospital",
```

Comment:
Should we put the valid types in an enum?

Also perhaps we'll need to add this to the org create form? None of the facilities we're currently working with are hospitals.